build.gradle

```diff
@@ -47,4 +47,43 @@ task writePom {
             }
         }.writeTo("$buildDir/pom.xml")
     }
+}
+
+task javadocJar(type: Jar) {
+    archiveClassifier = 'javadoc'
+    from javadoc
+}
+
+task sourcesJar(type: Jar) {
+    archiveClassifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+artifacts {
+    archives javadocJar, sourcesJar
+}
+
+// Task for Sonatype Nexus OSS: build, sign, and upload
+uploadArchives {
+    repositories {
+        mavenDeployer {
+            // Sign POM
+            beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+            // Destination
+            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+                authentication(userName: ossrhUsername, password: ossrhPassword)
```

Comment:
hmmm... placeholders?

Reply:
No, the package is pushed here, when the 'version' of the package does not contain '-SNAPSHOP' in the title. 
The package ends up in our private repo
<img width="1408" alt="Screen Shot 2020-02-07 at 2 33 10 PM" src="https://user-images.githubusercontent.com/34955647/74080511-93963300-49f9-11ea-83fe-dcdc91956831.png">